edge-modules/MetricsCollector/docker/windows/amd64/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG base_tag=2.1.13-nanoserver-1809
+FROM mcr.microsoft.com/dotnet/core/runtime:${base_tag}
+
+ARG EXE_DIR=.
+
```

Comment:
I'm pretty sure you must be container admin to scrape metrics on windows. @varunpuranik can you confirm?

Reply:
I confirmed this yesterday and will make this change.